account_banking_sepa_direct_debit/reports/sepa_direct_debit_mandate.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <template id="assets_common" name="account_banking_sepa_direct_debit pdf assets" inherit_id="web.report_assets_common">
```

Comment:
With this, you are overwriting the style for all the reports. In-line style is not working? If you want to do it this way, please put all the styles under a class for not changing general ones.

Reply:
@pedrobaeza you're right! I made the changes, could you check?